inc/html.class.php

```diff
@@ -3598,10 +3577,10 @@ static function initEditorSystem($name, $rand = '', $display = true, $readonly =
     *
     * @since 9.2
     *
-    * @param $content : content to convert in html
+    * @param mixed $content to convert in html
     *
-    * @return $content
-   **/
+    * @return mixed
```

Comment:
It has been made to be called using a string and return a string.

Reply:
Html::entity_decode_deep and Toolbox::convertImageToTag could return different values so you cant predict which is going to be the result, documenting the return as mixed solves its discrepancy.